src/js/pages/catalog/__tests__/PackageDetailTab-test.js

```diff
@@ -22,7 +22,7 @@ describe("PackageDetailTab", function() {
         location={{ query: { version: 1 } }}
       />,
       thisContainer
-    );
+    ).getWrappedInstance();
```

Comment:
We need this because of the withI18n. https://github.com/dcos/dcos-ui/pull/4076/files#diff-ef04d5bf3205041917c17a93e5389d86R688

Reply:
this is the first time we introduced this to our code. maybe we're doing something wrong here?